operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/dependent/DependentResourceConfigurationResolver.java

```diff
@@ -84,17 +84,8 @@ public static <C extends ControllerConfiguration<? extends HasMetadata>> Object
   private static ConfiguredClassPair getConfigured(
```

Comment:
Also tested this with Dependent resources with multiple `@Configured` annotations in the hierarchy, the result was the same for old and new implementation.

> If multiple classes in a hierarchy are annotated with an @Inherited annotation, only the annotation furthest from the base class is returned.

And this is exactly what the old implementation (manually) did, because it starts from the child class and goes up until it finds an annotation.

I did not commit this test, because it seems overkill.